packages/next/src/server/lib/start-server.ts

```diff
@@ -127,44 +132,7 @@ export interface StartServerOptions {
   keepAliveTimeout?: number
   // this is dev-server only
   selfSignedCertificate?: SelfSignedCertificate
-}
-
-export async function getRequestHandlers({
```

Comment:
This function was a bit silly, it's just passing everything through to `initialize`. The only thing it did was rename the `dev` argument to `isDev`. I'm just calling `initialize` directly now.